recipe/meta.yaml

```
@@ -81,7 +81,7 @@ requirements:
     - watchdog >=0.10.3
     - yarl >=1.9.4
   run_constrained:
-    - menuinst >=2.0.2
+    - menuinst >=2.1
```

```suggestion
    - menuinst >=2.1.2
```

This is the first version that supports file associations, right?

---

Correct. I'll also update the build in the Spyder repo.